CCDB/src/CcdbApi.cxx

```diff
@@ -1683,12 +1681,14 @@ void CcdbApi::vectoredLoadFileToMemory(std::vector<RequestContext>& requestConte
   // Save snapshots
   for (int i = 0; i < requestContexts.size(); i++) {
     auto& requestContext = requestContexts.at(i);
+    logReading(requestContext.path, requestContext.timestamp, &requestContext.headers,
+               fmt::format("{}{}", requestContext.considerSnapshot ? "load to memory" : "retrieve", fromSnapshots.at(i) ? " from snapshot" : ""));
     if (!requestContext.dest.empty()) {
       if (requestContext.considerSnapshot && fromSnapshots.at(i) != 2) {
         saveSnapshot(requestContext);
       }
     } else {
-      LOG(warning) << "Did not receive content for " << requestContext.path << "\n"; // Temporarily demoted to warning, since it floods the infologger
+      LOG(info) << "Did not receive content for " << requestContext.path << "\n"; // This is to be expected in case of http answer 304 and similar cases
```

Comment:
Cannot we suppress the message completely when the content is not supposed to be shipped?

Reply:
We can definitely do that in case of receiving http 304. I don't know if there are any other similar cases, but most likely that could be done.

Reply:
Only 304 is a legitimate case of not shipping an object, and we don't need to log this case. All other cases I know should produce a `LOG(error)`.

Reply:
Then I'll modify the code so it log's an error in case of non-304 response and no object received.

Reply:
In principle only http codes >= 400 should be considered errors and returned. Or libcurl IO errors, if the call could not be performed at all.